src/plugins/commander/commands/member.js

```diff
@@ -0,0 +1,61 @@
+const Command = require('../structs/Command.js');
+const got = require('got');
+
+class MembersCommand extends Command {
```

Comment:
Why plural?

Reply:
because I guess thats how I typed it when I did the thing, I'll change it to singular